site/content/docs/commands/server-mode.md

```diff
@@ -0,0 +1,14 @@
+# server-mode command
+
+### Usage
+    dotnet aws server-mode - Launches the tool in a server mode for integrations with IDEs, e.g. Visual Studio.
```

Comment:
nit: Internal technical writing guide recommends writing out "that is"/"for example"/etc. instead of "e.g." to avoid confusion for a global audience.

Down on line 10 too.

Reply:
done